DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -10991,8 +10991,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/DuckDuckGo/BrowserServicesKit";
 			requirement = {
-				kind = exactVersion;
-				version = 209.0.0;
+				branch = "graeme/speculative-password-import-prompt-crash-fix";
```

Comment:
Should not we wait to merge `graeme/speculative-password-import-prompt-crash-fix` into main branch and then refer the version, as previously done?

Reply:
@ankushkushwaha The `BrowserServicesKit` changes and the changes in this PR are dependent. It doesn't really make sense to create a new `BrowserServicesKit` release until it's all reviewed so, for now, pointing to a branch makes sense. Once the PR is approved, then of course we will point to a released `BrowserServicesKit` version.